testbed/todolist.test.js

```diff
@@ -0,0 +1,159 @@
+import  Application from '../modules/application/application.js';
+
+describe('Add to list', () => {
+    test('Test case 1', () => {
+        document.body.innerHTML =
+        '<div>' +
+        '  <ul class="todo-list"></ul>' +
+        '</div>';
+        
+        const Test1 = new Application(true);
+        Test1.addTaskToList("TaskTest_1 of Example 1");
+        Test1.addTaskToList("TaskTest_1 of Example-2");
+        Test1.addTaskToList("TaskTest_1 of Example-3");
+        const list = document.querySelectorAll('.todo-list li');
+        expect(list).toHaveLength(3);
+    });
+
+    test('Test case 2', () => {
+        document.body.innerHTML =
+        '<div>' +
+        '  <ul class="todo-list"></ul>' +
```

Comment:
- [ ] I would recommend using mock for functions that used localStorage to show that localstorage is also working fine.